scss/custom/_breadcrumb.scss

```diff
@@ -0,0 +1,14 @@
+.breadcrumb {
+  .breadcrumb-item {
+    color: $breadcrumb-item-color;
+    text-decoration: underline;
+
+    &.active {
+      text-decoration: none;
+    }
+  }
+
+  li.breadcrumb-item a {
```

Comment:
The breadcrumb items cannot only be nested inside a list element, it's also possible to use a-tags within a <nav> your style declaration should follow this idea.